### ISSUE TITLE
Possibly implementation for capital unblended Uk

### ISSUE THREAD

**Issue by KrasnayaPloshchad** [typiconman/fonts-cu#78]
On this _Biblija Ruska_ I saw capital unblended Uk.
https://en.wikipedia.org/wiki/File:Biblia_Ruska.jpg
Maybe you can create a new glyph in your fonts, and make it available via `cvXX`, `salt` or `ssXX` feature.

**Comment by typiconman** [comment]
Yes, but this should be added to a font that mimics the style of the Bibliya Ruska.
